packages/cypress/tests/cypress.config.ts

```diff
@@ -1,14 +1,23 @@
 import { defineConfig } from 'cypress';
 import { installPlugin } from '../dist';
+import { existsSync } from 'node:fs';
 
 export default defineConfig({
+  // `downloadsFolder` cannot be overridden in tests, so we're setting
+  // this to a non-default value for asserting in the tests
+  downloadsFolder: 'cypress/test-downloads',
```

Comment:
@skitterm how do you feel about setting the `downloadsFolder` to a non-default for all tests here? It's not allowed to be overridden in a single test.

Reply:
Ideally, we would have tests that cover both default and non-default settings. Would it be possible to achieve that here without great effort?

Reply:
@andrewortwein unfortunately, I think we'd need to write a second Cypress test suite with different configs.

Reply:
@tevanoff I think it's fine to do as you've done! I don't see a way in Cypress of overriding the `downloadsFolder` at the test level.

Reply:
(and it doesn't seem worth the complexity to have 2 Cypress test suites here, at least not yet)